apps/filebrowser/src/filebrowser/templates/listdir_components.mako

```diff
@@ -2489,7 +2489,7 @@ else:
 
 
       $("#editBreadcrumb").click(function (e) {
-        if ($(e.target).is('ul')){
+        if ($(e.target).is('ul') && !window.RAZ_IS_ENABLED){
```

Comment:
Can we still edit the breadcrumbs though?

i.e. we should be able to edit, just reset to current value if the bucket is changed

Reply:
done,